apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/ShardDownloader.java

```diff
@@ -263,7 +269,13 @@ public FileDownloadDecision tryDownloadShard(Long shardId) {
     public FileDownloadDecision prepareAndStartDownload() {
         boolean goodShardFound = false;
         log.debug("prepareAndStartDownload...");
+        boolean doNotShardImport = propertiesHolder.getBooleanProperty("apl.noshardimport", false);
         FileDownloadDecision result = FileDownloadDecision.NotReady;
+        if(doNotShardImport){
```

Comment:
Pls add log here with explanation why we go with Genesis data. It will be helpful later.